src/common/RAJAPerfSuite.cpp

```diff
@@ -20,6 +20,7 @@
 #include "basic/INIT_VIEW1D.hpp"
 #include "basic/INIT_VIEW1D_OFFSET.hpp"
 #include "basic/MULADDSUB.hpp"
+#include "basic/MAT_MAT_SHARED.hpp"
```

Comment:
Please move this line and the ones below before the MULADDSUB line. They are in alphabetical order based on kernel name to make it easier to see that everything is defined consistently.

Reply:
Fixed!